go.html.markdown

```diff
@@ -133,6 +135,11 @@ can include line breaks.` // Same string type.
 	// Unused variables are an error in Go.
 	// The underscore lets you "use" a variable but discard its value.
 	_, _, _, _, _, _, _, _, _, _ = str, s2, g, f, u, pi, n, a3, s4, bs
+	// Usually you use it to ignore one of the return values of a function
+	file, _ := os.Create("output.txt") // Should never ever cause an error
```

Comment:
The comment here is a bit confusing. Do you mean to say file creation never causes an error in general, or that if we assume this doesn't cause an error, it would be safe to ignore the error return value with `_`?